package.json

```diff
@@ -57,7 +57,7 @@
     "react": "16.0.0-alpha.12",
     "react-native": "0.48.1",
     "react-native-actionsheet": "^2.2.0",
-    "react-native-app-intro": "^1.1.5",
+    "react-native-app-intro": "https://github.com/Ernoff/react-native-app-intro.git#updated-packageJson",
```

Comment:
:clap: Though it would be sweet if you could publish on NPM if at all possible so we don't have too many GitHub repos in the `package.json`.

Reply:
Okay Great!! 
 I will get down to it